openshift/tests-extension/Makefile

```diff
@@ -89,7 +89,7 @@ build: bindata #HELP Build the extended tests binary
 	GO_COMPLIANCE_POLICY="exempt_all" go build -ldflags "$(LDFLAGS)" -mod=vendor -o $(TOOLS_BIN_DIR)/olmv1-tests-ext ./cmd/...
 
 .PHONY: update-metadata
-update-metadata: #HELP Build and run 'update-metadata' to generate test metadata
+update-metadata: build #HELP Build and run 'update-metadata' to generate test metadata
```

Comment:
`build` is needed there to build the `$(TOOLS_BIN_DIR)/olmv1-tests-ext` executable used on the next line...